src/components/views/rooms/RoomHeader.tsx

```diff
@@ -118,14 +123,33 @@ export default function RoomHeader({
 
     const videoClick = useCallback((ev) => videoCallClick(ev, callOptions[0]), [callOptions, videoCallClick]);
 
+    const shareClick = useCallback(() => {
+        const target = generateCallLink ? generateCallLink() : undefined;
+        if (target) {
+            Modal.createDialog(ShareDialog, {
+                target,
+                customTitle: _t("share|share_call"),
+                subtitle: _t("share|share_call_subtitle"),
+            });
+        } else {
+            logger.error("Could not generate call link.");
```

Comment:
Is there anything this could log that would give information on why it happened? Otherwise it doesn't feel very useful.